keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/KeanuRandomTest.java

```diff
@@ -0,0 +1,37 @@
+package io.improbable.keanu.vertices.dbl;
+
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class KeanuRandomTest {
+
+    @Test
+    public void canSampleGuassisnScalar(){
+        KeanuRandom random = new KeanuRandom(1);
+        DoubleTensor scalar = random.nextGaussian(new long[0]);
```

Comment:
you ought to test a [1] and a [1, 1] shape too

Reply:
👍